arch/x86/trusty.c

```diff
@@ -294,7 +295,8 @@ static bool setup_trusty_info(struct vcpu *vcpu,
 	/* TODO: prepare vkey_info */
 
 	/* copy key_info to the first page of trusty memory */
-	mem->first_page.key_info = g_key_info;
+	memcpy_s(&mem->first_page.key_info, sizeof(g_key_info),
```

Comment:
technically 2nd argument needs to be the size of the destination, not the source.